internal/db/schema/manager.go

```diff
@@ -244,57 +247,68 @@ func (b *Manager) runMigrations(ctx context.Context, p *provider.Provider) ([]Re
 	const op = "schema.(Manager).runMigrations"
 
 	var logEntries []RepairLog
-	var err error
```

Comment:
Sorry, I should've left a comment on this PR. The better way to do this is to use a named variable error return:
```go
func (b *Manager) runMigrations(ctx context.Context, p *provider.Provider) (_ []RepairLog, retErr error)
	...
	defer func() {
		if retErr != nil {
			...
		}
	}()
	...
}
```

This would allow us better guarantees that hoping that `errFinal` will always be assigned correctly before returning, because in deferred functions, `retErr` will always be equal to the error returned in any return statements (this is a guarantee of named return variables). So we can revert most of the `errFinal` stuff in this PR and just use a named return error (conventionally named `retErr`, there are other instances in the codebase).